benchmarks/versus-external-pools/package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "poolifier-benchmarks",
+  "version": "1.0.0-internal",
+  "description": "This project contains benchmarks for the most used and popular Node.js thread/cluster pools implementations",
+  "private": true,
+  "main": "index.js",
+  "author": "pioardi",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "dependencies": {
+    "benchmark": "^2.1.4",
+    "microjob": "0.7.0",
```

Comment:
I intentionally set all the direct pool dependencies to exact version matches, so it is clear which version was compared against each other

Reply:
It Is ok

Reply:
For now